### PR TITLE
feat: (KvProgressBar)  - Accessible progress bar with animation on load

### PR DESCRIPTION
**Pull request by BoulderBrains**
[vue-583](https://kiva.atlassian.net/browse/VUE-583)

KvProgressBar w/animation
- Included [accessibility classes](https://getbootstrap.com/docs/4.0/components/progress/)
	- :aria-valuemax="value"
	- :aria-valuenow="max"
	- aria-valuemin="0"
- Animation is timed over 1000ms with an ease-in

Direct storybook link, toggle between the 2 variants to replay the animation.
https://608b4cf87f686c00213841b1-xgsjubxhfh.chromatic.com/?path=/story/kvprogressbar--variant-brand

### CODE REVIEW

File: kv-components/vue/KvProgressBar.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<template>
+	<div
+		role="progressbar"
+		aria-label="progress meter"
+		class="h-1 w-full rounded-full bg-gray-300 bg-opacity-low overflow-hidden"
+	>
+		<div
+			ref="progressBarRef"
```
Comment (eddieferrer) [inline kv-components/vue/KvProgressBar.vue:8]: Is this needed?

File: kv-components/vue/stories/KvProgressBar.stories.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		},
+	},
+};
+
+const Template = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { KvProgressBar },
+	template: `
+		<kv-progress-bar
+			ref="progressBarRef"
```
Comment (eddieferrer) [inline kv-components/vue/stories/KvProgressBar.stories.js:26]: Is this needed?

File: kv-components/vue/KvProgressBar.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+			:max="max"
+			:value="value"
+			class="h-1 rounded-full transition-all duration-1000 origin-left ease-in bg-brand"
+			:style="{width: loaded ? `${value}%` : '0' }"
+			:aria-valuemax="value"
+			:aria-valuenow="max"
```
Comment (emuvente) [inline kv-components/vue/KvProgressBar.vue:14]: It looks like valuemax and valuenow are swapped

Comment (BoulderBrains) [reply kv-components/vue/KvProgressBar.vue]: Good catch, thank you, I've corrected this in my latest commit.

File: kv-components/vue/stories/KvProgressBar.stories.js (hunk 2 above)
Comment (BoulderBrains) [reply kv-components/vue/stories/KvProgressBar.stories.js]: Thanks for catching this Eddie, it's been removed in the latest commit.

File: kv-components/vue/KvProgressBar.vue (hunk 1 above)
Comment (BoulderBrains) [reply kv-components/vue/KvProgressBar.vue]: nope, removed in latest commit.

**Review by eddieferrer — APPROVED** [review]
Looks great 👍

**Review by emuvente — APPROVED** [review]
👍 nice!

**Review by mcstover — APPROVED** [review]

**Pull request merged**